src/commands/cluster-slot-stats.json

```
@@ -0,0 +1,91 @@
+{
+    "SLOT-STATS": {
+        "summary": "Return an array of slot usage statistics for slots assigned to the current node.",
+        "complexity": "O(N) where N is the total number of slots based on arguments. O(N*log(N)) with ORDERBY subcommand.",
+        "group": "cluster",
+        "since": "8.0.0",
+        "arity": -4,
+        "container": "CLUSTER",
+        "function": "clusterSlotStatsCommand",
+        "command_flags": [
+            "STALE",
+            "LOADING"
+        ],
+        "command_tips": [
+            "NONDETERMINISTIC_OUTPUT",
+            "REQUEST_POLICY:ALL_SHARDS"
+        ],
+        "reply_schema": {
+            "type": "array",
+            "description": "Array of nested arrays, where the inner array element represents a slot and its respective usage statistics.",
+            "items": {
+                "type": "array",
+                "description": "Array of size 2, where 0th index represents (int) slot, and 1st index represents (map) usage statistics.",
+                "items": {
+                    "type": "object",
+                    "description": "Map of slot usage statistics.",
+                    "additionalProperties": {
+                        "type": "string"
+                    }
+                }
+            }
+        },
```

```suggestion
        "reply_schema": {
            "type": "array",
            "description": "Array of nested arrays, where the inner array element represents a slot and its respective usage statistics.",
            "items": {
                "type": "array",
                "description": "Array of size 2, where 0th index represents (int) slot and 1st index represents (map) usage statistics.",
                "minItems": 2,
                "maxItems": 2,
                "items": [
                    {
                        "description": "Slot Number.",
                        "type": "integer"
                    },
                    {
                        "type": "object",
                        "description": "Map of slot usage statistics.",
                        "additionalProperties": {
                            "type": "string"
                        }
                    }
                ]
            }
        },
```

This seems more right? I'm going to kick off the protocol parser on this, since I think it will complain about it.